aip/0160.md

```diff
@@ -103,12 +114,13 @@ translate the string to the appropriate strongly-typed value:
 **Warning:** The identifiers `true`, `false`, and `null` only carry intrinsic
 meaning when used in the context of a typed field reference.
 
-Additionally, when comparing strings for equality, the `*` character denotes
-wildcard syntax; for example, `a = "*.foo"` is true if `a` _ends with_ ".foo".
+Additionally, when comparing strings for equality, services **should** support
+wildcards using the `*` character; for example, `a = "*.foo"` is true if `a`
```

Comment:
is this intended as a suffix/prefix match only or general wildcard support ?  comments on the list filtering design doc seem to suggest only allowing * as a suffix ( i.e. allow only prefix matching ).

Reply:
It seems fine to do more, but we probably only want to require prefix/suffix because of subtleties around values with `*` in them.